main/io.home-assistant/battery.js

```diff
@@ -0,0 +1,35 @@
+// -*- mode: js; indent-tabs-mode: nil; js-basic-offset: 4 -*-
+//
+// This file is part of io.home-assistant
+//
+// Copyright 2021
+//
+// See LICENSE for details
+"use strict";
+
+const HomeAssistantDevice = require('./base');
+
+module.exports = class HomeAssistantBattery extends HomeAssistantDevice {
+    constructor(engine, state, master, entityId) {
+        super(engine, state, master, entityId);
+        const [domain,] = entityId.split('.');
+		this.domain = domain;
```

Comment:
Looks like the indent is wrong, please set your editor to use 4 spaces everywhere

Reply:
I will check